[2015-05-27 19:00:00][ERROR][testlog.py:<module>:8] aaa
